sunpy/net/jsoc/jsoc.py

```diff
@@ -432,18 +439,25 @@ def request_data(self, jsoc_response, **kwargs):
 
         requests = []
         self.query_args = jsoc_response.query_args
+        supported_protocols = {'fits', 'as-is'}
+        supported_methods = {'url-tar', 'url', 'url-quick'}
         for block in jsoc_response.query_args:
 
             ds = self._make_recordset(**block)
             cd = drms.Client(email=block.get('notify', ''))
             protocol = block.get('protocol', 'fits')
```

Comment:
We don't have to fix this here, but it feels weird that email, protocol and notify are not all the same?